source/fuzz/fuzzer_util.cpp

```diff
@@ -759,6 +759,51 @@ opt::Function* GetFunctionFromParameterId(opt::IRContext* ir_context,
   return nullptr;
 }
 
+uint32_t MaybeReuseFunctionType(
```

Comment:
I would appreciate a tad more commentary in this method body.  Not tons of comments, but a few high level ones to explain what is going on.

Reply:
Fixed in e5fda6e.